docs/source/notebooks/08_Getting_and_bias_correcting_CMIP6_data.ipynb

```diff
@@ -0,0 +1,1788 @@
+{
```

Comment:
<p>Nitpicking, but I think end_day would be closer to what you mean than stop_day.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/raven/pull/424/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Good point, changed.</p>